src/test/resources/com/puppycrawl/tools/checkstyle/indentation/InputAnnotationClosingBracketEndInSameIndentationAsOpening.java

```diff
@@ -0,0 +1,26 @@
+package com.puppycrawl.tools.checkstyle.indentation; //indent:0 exp:0
+
+/**                                                                           //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * arrayInitIndent = 4                                                        //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * braceAdjustment = 0                                                        //indent:1 exp:1
+ * caseIndent = 4                                                             //indent:1 exp:1
+ * forceStrictCondition = false                                               //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ * throwsIndent = 4                                                           //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+
+@interface SimpleType { //indent:0 exp:0
+    Class<?> value(); //indent:4 exp:4
+} //indent:0 exp:0
+
+@SimpleType( //indent:0 exp:0
+    value = Boolean.class //indent:4 exp:4
+) //indent:0 exp:0
+public class InputAnnotationClosingBracketEndInSameIndentationAsOpening { //indent:0 exp:0
+} //indent:0 exp:0
```

Comment:
This input is about `AnnotationClosingBracketEndInSameIndentationAsOpening`. For sure we can think of some more annotations for this file proving that functionality works correctly, e.g. something like that:
```java
@SimpleType( value = Boolean.class
) 
```